### PR TITLE
Adds Slack Integration

### PR DESCRIPTION
**Pull request by thisdarktao**
Task updates can now be broadcasted to Slack channels and users as well as regular email.

### CODE REVIEW

File: app/tasks.py [hunk 1]
```diff
@@ -1,53 +1,112 @@
...
 
-    to_addresses = []
-    if to:
-        to_addresses.append(to)
-
     notify_job_relationships = task.status.notify_job_relationships
     notify_task_assignees = task.status.notify_task_assignees
 
+    print(notify_job_relationships, notify_task_assignees)
+
```
Comment (stuartaccent) [inline app/tasks.py]: boooooo :)

File: common/admin.py [hunk 2]
```diff
@@ -0,0 +1,4 @@
+from django.contrib import admin
+
+from .models import WIPSettings
```
Comment (stuartaccent) [inline common/admin.py]: needs 2 rows after imports

File: common/apps.py [hunk 3]
```diff
@@ -0,0 +1,5 @@
+from django.apps import AppConfig
+
+
+class CommonConfig(AppConfig):
+    name = 'common'
```
Comment (stuartaccent) [inline common/apps.py]: loose this file

File: common/views.py [hunk 4]
```diff
@@ -0,0 +1,3 @@
+from django.shortcuts import render
+
+# Create your views here.
```
Comment (stuartaccent) [inline common/views.py]: loose this file

File: frontend/src/app/forms/job-detail.form.ts [hunk 5]
```diff
@@ -49,6 +50,7 @@ export class JobDetailForm extends BaseForm {
                 colour: new FormControl('', [Validators.required, Validators.pattern(hexReg)]),
                 status: new FormControl('', Validators.required),
                 billed_to: new FormControl(null),
+                slack_channel_id: new FormControl('', [Validators.minLength(9), Validators.maxLength(9)])
```
Comment (craigvincent) [inline frontend/src/app/forms/job-detail.form.ts]: Is this validator right dude? min length 9 / max length 9? Just checking :)

Comment (thisdarktao) [reply frontend/src/app/forms/job-detail.form.ts]: Yeah the channel IDs are always 9 characters in length 👍

File: app/tasks.py (hunk 1 above)
Comment (thisdarktao) [reply app/tasks.py]: 😭

**Pull request merged**